src/main/java/com/xebia/incubator/xebium/DefaultWebDriverSupplier.java

```diff
@@ -126,6 +126,10 @@ public void setBrowser(String browser) {
 		this.browser = browser;
 	}
 
+    public String getBrowser() {
```

Comment:
Please use tabs for indentations (though I realize we're being inconsistent about that, #147)